Packs/QRadar/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,11 @@
+
+#### Incident Types
+- **Qradar Generic**
+Assigned the **QRadar Generic** playbook as the default playbook for this incident type.
+
+#### Playbooks
+##### QRadar Generic
+The default playbook for working with the QRadar V2 integration for basic SIEM/SOC operations.
+#### Classifiers
+##### QRadar - Generic Incoming Mapper
+- Added mapping for the severity field bases on the magnitude value. The XSOAR incident severity field values range is 0-4 where<br/>0 - Informational<br/>1 - Low<br/>2 - Medium<br/>3 - High<br/>4 - Critical<br/><br/>A scale is required to translate the value of the magnitude field value to a valid incident severity value. The default scale is 1,1,1,2,2,2,2,3,3,3 <br/>The meaning of the default scale is that values 1-3 of magnitude will be translated to low severity \(positions 1-3 in the scale\), values 4-7 will be translated to medium severity \(positions 4-7 in the scale\) and values 8-10 will be translated to high severity \(positions 8-10 in the scale\).
```

Comment:
@Michal-Vardi And this release note which refers to the change in the mapper and not the playbook